static/downloadables/grep_file.py

```diff
@@ -348,7 +354,10 @@ def shouldUseAssociation(traits, studyIDs, studyTypes, ethnicities, sexes, value
     useValueType = False
     useSex = False
 
-    studyValueTypes = set([x.split("|")[-1] for x in studyMetaData['traits'][trait]['pValBetaAnnoValType']])
+    if studyMetaData is not None:
```

Comment:
Why this change?

Reply:
I was running into some cases where the studyMetaData was None and so the studyValueTypes initiation was throwing an error.

Reply:
When was it None? if it is none, then we most likely have a problem

Reply:
Oh, I wrote that code before I put in the check to make sure users weren't specifying any study filters if they were also uploading their own GWAS data. We system exit with an explanation now, but before we were setting the studyMetaData to None if the studyID wasn't in the database and then still trying to get the studyBalueTypes from it. I'm going to remove the if statement, though, because that will no longer happen.